sklearn/cluster/tests/test_feature_agglomeration.py

```diff
@@ -0,0 +1,39 @@
+"""
+Tests for sklearn.cluster._feature_agglomeration
+"""
+# Authors: Sergul Aydore 2017
+import numpy as np
+from sklearn.cluster import FeatureAgglomeration
+from sklearn.utils.testing import assert_true
+from sklearn.utils.testing import assert_array_almost_equal
+
+
+def test_feature_agglomeration():
+    n_clusters = 1
+    X = np.array([0, 0, 1], ndmin=2)  # (n_samples, n_features)
```

Comment:
Could you write this out in full as nested lists, or reshape so that the orientation is clearer to the reader, pease?